src/plugins/imagery/components/Compass/CompassRose.vue

```diff
@@ -177,12 +185,28 @@ export default {
         }
     },
     computed: {
-        north() {
-            return this.lockCompass ? rotate(-this.cameraPan) : 0;
+        compassRoseSizing() {
```

Comment:
Can we name this something like compassRoseSizingClasses()? I feel like the current name implies it's going to do something.

Reply:
Got it, fixed.